infrastructure/resource_group_level/main.bicep

```diff
@@ -145,6 +146,7 @@ var dtfsCentralApiAdditionalSecureSetting = {
 
 var portalApiSettings = {
   RATE_LIMIT_THRESHOLD: RATE_LIMIT_THRESHOLD
+  PORTAL_UI_URL: PORTAL_UI_URL
```

Comment:
Note that because we are in control of the front-door deployment, we can actually get the value ourselves and avoid the GitHub variable. If there wasn't the issue of selecting the right url for the `prod` deployment I would insist on following the same non-trivial mechanism used by the `tfmUiUrl` setting. This is a little fiddly though as you need to avoid a circular dependency and set the value _after_ the front-door deployment. (Weirdly, the tfmUiUrl uses a connection string too.)

Don't do that now, but raise a card under FN-692 to at least consider it. Use FN-927 (The TFM_UI_URL one) as a template for it. Grab me if that's not clear!

Reply:
See FN-1479